packages/commands/generate/src/index.ts

```diff
@@ -161,24 +164,24 @@ export const plugin: CliPlugin = {
             new GithubLoader(),
           ], join(cwd, generateConfig.folders.model + '/**/*.graphql'));
 
+          const schemaString = printSchemaWithDirectives(models);
           const backendCreator = new GraphQLBackendCreator(
-            printSchemaWithDirectives(models),
+            {
+              getSchemaText: () => { return schemaString }
```

Comment:
If Graphback uses `GraphQLSchema` or `DocumentNode`, we can send it directly instead of parsing and converting it to string again and again.

Reply:
Yes. This is very early version. We are going to address that.
CC @craicoverflow

Reply:
Yeah this is something we are working on right now!

Reply:
Great! :)